src/components/BookItem.jsx

```diff
@@ -1,17 +1,25 @@
-import React from 'react';
+import React, { useEffect } from 'react';
 import { useSelector, useDispatch } from 'react-redux';
-import { removeBook } from '../redux/books/books';
+import { getBooks, removeBook } from '../redux/books/books';
 
 function BookItem() {
   const bookStore = useSelector((state) => state.booksReducer);
   const dispatch = useDispatch();
 
+  useEffect(() => {
+    if (!bookStore.length) {
+      dispatch(getBooks());
+    }
+  // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
As much as possible, kindly avoid disabling linter checks in-line. Using the quick fix option of updating the dependency array to `[bookStore.length, dispatch]` will resolve the warning.